tika-core/src/main/java/org/apache/tika/detect/TSDDetector.java

```diff
@@ -0,0 +1,80 @@
+//package it.notartel.cons.verifyfiles.custom;
+package org.apache.tika.detect;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+
+import org.apache.tika.detect.Detector;
+import org.apache.tika.io.LookaheadInputStream;
+import org.apache.tika.metadata.Metadata;
+import org.apache.tika.mime.MediaType;
+//import org.slf4j.Logger;
```

Comment:
Removed unnecessary comments and commented imports, please. 

If you nead logging in `tika-core`, you could use `java.util.logging` classes. In all other Tika packages you could use SLF4J with `org.slf4j.Logger` saved in `private static final Logger LOG` field.